Gemfile

```diff
@@ -6,7 +6,7 @@ gem 'devise'
 gem 'pg'
 gem 'puma', '~> 3.0'
 gem 'sass-rails', '~> 5.0'
-gem 'uglifier', '>= 1.3.0'
+gem 'uglifier', '>= 3.2.0'
```

Comment:
Sound good, the actual gem installed is 4.2.0, so this restriction can be `~> 4.2`